### PR TITLE
Add cmd reference & node management docs

### PR DESCRIPTION
**Pull request by jakolehm**

### CODE REVIEW

File: docs/node-management.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+Adding manager nodes is as simple as adding them into the `cluster.yaml`. Re-running `launchpad apply ...` will configure the control plane on the new node and also makes necessary changes in the swarm & etcd cluster.
+
+### Removing Manager Nodes
+
+Once you've determined that it is safe to remove a manager node, and its etcd peer, follow this process:
+
+1. Stop the node
+2. Update `cluster.yaml`
+3. Run `launchpad apply ...`
```
Comment (jnummelin) [inline docs/node-management.md]: This will be no-op in this case

File: docs/node-management.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+4. Terminate/remove the node in your infrastructure
```
Comment (jnummelin) [inline docs/node-management.md]: _someone_ should run `swarm leave ...` on the node too. Launchpad is not currently able to do that, so maybe mention that it's a user action.

File: docs/node-management.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+Raft requires a majority of managers, also called the quorum, to agree on proposed updates to the swarm, such as node additions or removals. Membership operations are subject to the same constraints as state replication.
+
+Keep in mind that the manager nodes also host the control plane etcd cluster. Even more importantly, any changes to the cluster require a working etcd cluster with the majority of peers present and working.
+
+As normally with quorum based systems, it is highly advisable to run an odd number of peers. As the control plane only works when a majority can be formed, once you grow the control plane to have more than one node, you can't (automatically) go back to having only one node.
+
+### Adding Manager Nodes
+
+Adding manager nodes is as simple as adding them into the `cluster.yaml`. Re-running `launchpad apply ...` will configure the control plane on the new node and also makes necessary changes in the swarm & etcd cluster.
```
Comment (jnummelin) [inline docs/node-management.md:19]: Maybe add a note that in this case managers will need to "rebalance" and re-cluster stuff so there might be some period of redness.

File: docs/node-management.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+### Adding Worker Nodes
+
+Adding worker nodes is as simple as adding them into the `cluster.yaml`. Re-running `launchpad apply ...` will configure everything on the new node and joins it into the cluster.
+
+### Removing Worker Nodes
+
+Removing a worker node is currently a multi step process:
+
+1. Remove the host from `cluster.yaml`. As re-applying a cluster would not actually do anything, you do not need to run `launchpad apply ...`
+2. Move workloads away from the node with draining
```
Comment (jnummelin) [inline docs/node-management.md]: And run `docker swarm leave ...`

**Review by jnummelin — DISMISSED** [review]
Need bit of finetuning for node removal parts as there's currently some limitations in launchpad for this.

**Review by nevalla — APPROVED** [review]
Besides Jussi's comments, LGTM

**Pull request merged**